src/components/home/sections/RelatedCards.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
hm, why is this being added again? did you move it? notice how it's hard for me to review this diff because it seems like the file is being added, even though basil had merged it earlier

Reply:
Actually I took the files in **temp** branch as they are, where my teammates created them, then I just applied the new configurations (prettier, eslint, and circleci) on them.